assets/docs/about-text.md

```diff
@@ -1,9 +1,19 @@
-The Deltares Global Data Services platform has been developed by Deltares as a free, web-based application to support the study and sharing of integrated water and subsoil-related data. The platform provides indicative data, that is primarily intended for professional specialists and researchers.
+This is a beta release of the Blue Earth Data platform. The Blue Earth Data platform has been developed by Deltares as a free, web-based application to support the study and sharing of integrated water and subsoil-related data. The platform provides indicative data, which is primarily intended for professional specialists and researchers.  
+&nbsp;
```

Comment:
Are all these no-break spaces necessary? If it's just to create extra whitespace we can solve this by changing the styling in CSS.

Reply:
If we can solve this by changing the styling in the CSS that would be fine as well. I just modified the text. Most of the no-break spaces were already in there. So maybe also check with Cindy. Also note that if changes to the CSS are required, this is probably up to you or Cindy (to do it well).

Reply:
Ok, then we will increase spacing in the styling.

Reply:
Should I commit modified version of these files without the no-break spaces in this case? Or will you also modify this?

Reply:
If it's easy, yes please!

Reply:
Nog ever ter bevestiging: geldt dit voor zowel user-agreements.md, cookie-agreements.md en about-text.md? En kunnen alle no-break spaces uit deze files verwijderd worden?

Reply:
Yes, de styling geldt voor alle markdown teksten

Reply:
Ik heb de no-break spaces uit de bestanden verwijderd.